src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6439,13 +6452,13 @@ s32 func_8083F360(PlayState* play, Player* this, f32 arg1, f32 arg2, f32 arg3, f
     sp60.y = sp6C.y = this->actor.world.pos.y + arg1;
 
     if (BgCheck_EntityLineTest1(&play->colCtx, &sp6C, &sp60, &sp54, &this->actor.wallPoly, true, false, false, true,
-                                &sp78)) {
+                                &wallBgId)) {
         wallPoly = this->actor.wallPoly;
 
         this->actor.bgCheckFlags |= BGCHECKFLAG_PLAYER_WALL_INTERACT;
-        this->actor.wallBgId = sp78;
+        this->actor.wallBgId = wallBgId;
 
-        D_808535F0 = func_80041DB8(&play->colCtx, wallPoly, sp78);
+        sWallFlags = SurfaceType_GetWallFlags(&play->colCtx, wallPoly, wallBgId);
```

Comment:
possibly `sInteractWallFlags`? makes it more clear its the flags of the wall youre interacting with. and is also set with `BGCHECKFLAG_PLAYER_WALL_INTERACT`